### DISCUSSION TITLE
Windows install via Store or Certified vendor

### DISCUSSION THREAD

**Discussion by jfmatth** [containers/podman#15603] [Ideas]
I love Podman, and now with the Remote client for Windows integration with WSL, it's even better.

However, would be great to have it in the Windows Store.  If not that, then at least have it so Windows doesn't think it's bad software, with tons of warnings, etc.

Not sure how to do either of those, but would be great to see Podman as a 1st class citizen in the Windows realm.

**Comment by mheon** [comment]
@n1hility may have thoughts here

**Comment by n1hility** [comment]
@jfmatth A new signed installer with trusted behavior is on it's way:
https://github.com/containers/podman/pull/15610

We aren't yet looking at the windows store, but we are looking to put podman-desktop (includes podman) in the main package managers (winget, chocolatey)
